test/pkg_finish_test.rb

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+require "packager/clients/pkg_finish"
+require "packages/repository"
+require "uri"
+require "tmpdir"
+require "fileutils"
+
+describe Yast::PkgFinishClient do
+  Yast.import "Pkg"
+  Yast.import "Installation"
+  Yast.import "WFM"
+
+  FAILED_PKGS_PATH = "/var/lib/YaST2/failed_packages"
+
+  subject(:client) { Yast::PkgFinishClient.new }
```

Comment:
You're right. But to be honest I usually prefer the use of [named subjects](http://blog.davidchelimsky.net/blog/2012/05/13/spec-smell-explicit-use-of-subject/).